Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/FilterScreen.kt

```diff
@@ -0,0 +1,261 @@
+                        onChanged = {
+                            sortChanged = it
+                        }
+                    )
+                }
+                FilterTitle(text = stringResource(id = R.string.price))
+                Row(
+                    horizontalArrangement = Arrangement.Center,
+                    modifier = Modifier.fillMaxWidth()
+                ) {
+                    priceFilters.forEach {
+                        FilterChip(
+                            filter = it,
+                            modifier = Modifier.padding(end = 5.dp)
+                        )
+                    }
+                }
+                FilterTitle(text = stringResource(id = R.string.category))
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(top = 12.dp, bottom = 28.dp),
+                    horizontalArrangement = Arrangement.Center
+                ) {
+                    FlowRow(
+                        modifier = Modifier.padding(horizontal = 4.dp)
+                    ) {
+                        categoryFilters.forEach {
+                            FilterChip(
+                                filter = it,
+                                modifier = Modifier.padding(end = 5.dp, bottom = 8.dp)
+                            )
+                        }
+                    }
+                }
+                Row {
+                    FilterTitle(text = stringResource(id = R.string.max_calories))
+                    Text(
+                        text = stringResource(id = R.string.per_serving),
+                        style = MaterialTheme.typography.body2,
+                        color = JetsnackTheme.colors.brand,
+                        modifier = Modifier.padding(top = 5.dp, start = 10.dp)
+                    )
+                }
+                MaxCaloriesSlider()
+                FilterTitle(text = stringResource(id = R.string.lifestyle))
+                Row(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .padding(top = 12.dp, bottom = 28.dp),
+                    horizontalArrangement = Arrangement.Center
+                ) {
+                    FlowRow(
+                        modifier = Modifier.padding(horizontal = 4.dp)
+                    ) {
+                        lifeStyleFilters.forEach {
+                            FilterChip(
+                                filter = it,
+                                modifier = Modifier.padding(end = 5.dp, bottom = 8.dp)
+                            )
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun SortFilters(onChanged: (Boolean) -> Unit) {
+    val sortFilters = SnackRepo.getSortFilters()
+    val defaultValue = SnackRepo.getSortDefault()
+    var sortState by remember { mutableStateOf(SnackRepo.getSortDefault()) }
```

Comment:
Can we pass this in to hoist the sort state?

Reply:
Done